### DISCUSSION TITLE
Adding a field to an existing view "view.AddViewFieldAsync()" method not available.

### DISCUSSION THREAD

**Discussion by NewGHUser4321** [pnp/pnpcore#1282] [Q&A] [answered]
Can anyone please confirm if you see the method `AddViewFieldAsync()` available on a view as mentioned in the documentation [here](https://pnp.github.io/pnpcore/using-the-sdk/lists-views.html#adding-a-field-to-an-existing-view). This method is not available for me. 
SDK version: 1.8.0

Thank you.

**Comment by jansenbe** [comment, marked as answer]
@NewGHUser4321 : this was added with version 1.10:

![image](https://github.com/pnp/pnpcore/assets/7451219/65b9960a-2e4d-4983-8233-4719bba1daf1)

**Reply by NewGHUser4321** [reply]
@jansenbe Will check it out. Thank you.
